tools/pipelines/build-client.yml

```diff
@@ -13,6 +13,8 @@ parameters:
   values:
     - none
     - prerelease
+    - prerelease - alpha
+    - prerelease - beta
```

Comment:
These should just be `alpha` or `beta`.

Reply:
I thought could be a confusion whether it is "release" alpha or "prerelease" alpha. 

nvm, but our alpha and beta will always fall into prerelease.